test/authentication.js

```diff
@@ -13,31 +13,24 @@ describe("Testing jsonapi-client", function() {
         }
       });
 
-      client.get("people", { }, function(err) {
+      client.get("people", "ad3aa89e-9c5b-4ac9-a652-6670f9f27587", function(err) {
         assert.equal(err.message, "401 Unauthorized");
         done();
       });
     });
 
-    it("is denied access with the blockMe cookie", function(done) {
+    xit("is denied access with the blockMe cookie", function(done) {
```

Comment:
Yep - it doesn't work with CORS due to attempting to pass a cookie cross-domain (the Karma test suite runs on port 8124(ish) and the API runs on port 16006). Without specific CORS headers this will never work. It does however work server-to-server, but we're now running tests both sides.